komodo/build.py

```diff
@@ -115,17 +115,31 @@ def rsync(pkg, ver, pkgpath, data, prefix, *args, **kwargs):
     shell(['rsync -am', kwargs.get('makeopts'), '{}/'.format(pkgpath), kwargs['fakeroot'] + prefix])
 
 
-def pip_install(pkg, ver, pkgpath, data, prefix, dlprefix, pip='pip', *args, **kwargs):
-    cmd = [pip,
-           'install {}=={}'.format(pkg, komodo.strip_version(ver)),
-           '--root {}'.format(kwargs['fakeroot']),
-           '--prefix {}'.format(prefix),
-           '--no-index',
-           '--no-deps',
-           '--ignore-installed',
-           '--cache-dir {}'.format(dlprefix),
-           '--find-links {}'.format(dlprefix),
-           kwargs.get('makeopts', ''),
+def pip_install(
+    pkg,
+    ver,
+    pkgpath,
+    data,
+    prefix,
+    dlprefix,
+    pip="pip",
+    package_name=None,
```

Comment:
For `pip`, `pkg == package_name`, so drop this? Use `pkg` from this point on.

Reply:
This has now been changed